tests/test_create.py

```diff
@@ -2457,7 +2457,7 @@ def test_conda_downgrade(
         PrefixData._cache_.clear()
         subprocess_call(
             [
-                python_exe,
+                sys.executable,
```

Comment:
@kenodegard, can you confirm this fix is correct? Judging by the comment above, I think so? This makes the test pass.